java/vector/src/main/java/org/apache/arrow/vector/BitVector.java

```diff
@@ -126,7 +126,12 @@ public void setInitialCapacity(int valueCount) {
    */
   @Override
   public int getValueCapacity() {
-    return capAtMaxInt(validityBuffer.capacity() * 8);
+    return actualValueCapacity;
+  }
```

Comment:
The implementation is identical to that of the super class, so we have no need of overriding it?

Reply:
Thanks! Missed that. Removed!